src/components/pages/PendingApplications/PendingApplication.js

```diff
@@ -74,6 +74,18 @@ const PendingApplications = () => {
                 Date(row.created_at.slice).slice(10, 16),
               button: (
                 <Button
+                  style={{
+                    backgroundImage:
+                      'linear-gradient(-180deg, #37AEE2 0%, #1E96C8 100%)',
+                    borderRadius: '.5rem',
+                    boxSizing: 'border-box',
+                    color: '#FFFFFF',
```

Comment:
Black button on a blue gradient background? I feel as though that'll be super hard to read for anyone. Maybe something a little better on the eye?